compiler/src/dotty/tools/dotc/core/ConstraintHandling.scala

```diff
@@ -20,6 +20,9 @@ import config.Printers.{constr, typr}
  */
 trait ConstraintHandling {
 
+  def constr_println(msg: => String): Unit = constr.println(msg)
```

Comment:
I did this because without it, reading the printer output for GADTs was confusing even for small examples, since TyperState Constraint printed very similar messages.